Common/Product/SharedProject/ProjectNode.cs

```diff
@@ -1575,9 +1577,11 @@ internal enum EventTriggering {
                             }
 
                             FinishProjectCreation(basePath, baseLocation);
+                            this.LogProjectLoadTelemetry(TelemetryEvents.ProjectCreated);

```

Comment:
Removing this telemetry point altogether as existing VS telemetry already has it covered at a more general level and with more rich data.